test/TestHelpers.cs

```diff
@@ -275,7 +257,7 @@ private static List<string> GetLogs_Activity_Gets_HttpManagementPayload(string i
                     $"{instanceId}: Function '{functionNames[0]} ({FunctionType.Orchestrator})' started. IsReplay: False. Input: null. State: Started. HubName: {taskHubName}. AppName: . SlotName: . ExtensionVersion: 1.4.1.0. SequenceNumber: 1.",
                     $"{instanceId}: Function '{functionNames[1]} ({FunctionType.Activity})' scheduled. Reason: ReturnHttpManagementPayload. IsReplay: False. State: Scheduled. HubName: {taskHubName}. AppName: . SlotName: . ExtensionVersion: 1.4.1.0. SequenceNumber: 2.",
                     $"{instanceId}: Function '{functionNames[0]} ({FunctionType.Orchestrator})' awaited. IsReplay: False. State: Awaited. HubName: {taskHubName}. AppName: . SlotName: . ExtensionVersion: 1.4.1.0. SequenceNumber: 3.",
-                    $"{instanceId}: Function '{functionNames[1]} ({FunctionType.Activity})' started. IsReplay: False. Input: [{{\"$type\":\"Microsoft.Azure.WebJobs.Extensions.DurableTask.HttpManagementPayload, Microsoft.Azure.WebJobs.Extensions.DurableTask\",\"id\":\"{instanceId}\",\"statusQueryGetUri\":\"{TestConstants.NotificationUrlBase}/instances/{instanceId}?taskHub={taskHubName}&connection=Storage&code=mykey\",\"sendEventPostUri\":\"{TestConstants.NotificationUrlBase}/instances/{instanceId}/raiseEvent/{{eventName}}?taskHub={taskHubName}&connection=Storage&code=mykey\",\"terminatePostUri\":\"{TestConstants.NotificationUrlBase}/instances/{instanceId}/terminate?reason={{text}}&taskHub={taskHubName}&connection=Storage&code=mykey\"}}]. State: Started. HubName: {taskHubName}. AppName: . SlotName: . ExtensionVersion: 1.4.1.0. SequenceNumber: 4.",
+                    $"{instanceId}: Function '{functionNames[1]} ({FunctionType.Activity})' started. IsReplay: False. Input: [null]. State: Started. HubName: {taskHubName}. AppName: . SlotName: . ExtensionVersion: 1.4.1.0. SequenceNumber: 4.",
```

Comment:
Let's remove all the text at `HubName` and beyond. We don't want to test for this because that will cause test failures when, for example, we update the extension version. The test code does a prefix comparison, so the full trace doesn't need to match.